src/main/kotlin/no/nav/syfo/infrastructure/database/repository/VedtakRepository.kt

```diff
@@ -309,6 +342,13 @@ class VedtakRepository(private val database: DatabaseInterface) : IVedtakReposit
                  WHERE v.journalpost_id IS NULL
             """
 
+        private const val GET_NOT_OPPGAVE_VEDTAK =
+            """
+                 SELECT v.*
+                 FROM vedtak v
+                 WHERE v.journalpost_id IS NOT NULL AND v.journalpost_id != '0' AND v.oppgave_id IS NULL
```

Comment:
Tanken er at vi ikke lager oppgave før vedtaket er journalført.

Reply:
Enig i logikken 👍🏼 Trenger vi å navngi tabellen `v` her?
Er `v.journalpost_id != '0'` for å unngå dev-problemet med testpersoner uten aktørid?

Reply:
sql'en kan forenkles litt ja.

Ja, `v.journalpost_id != '0'` er for å unngå at vi sender oppgave på vedtakene som ikke kunne journalføres i dev.